src/db/tables/subscribers.js

```diff
@@ -271,8 +272,60 @@ async function deleteResolutionsWithEmail (id, email) {
 }
 /* c8 ignore stop */
 
+/**
+ * @param {Partial<{ plusOnly: boolean; limit: number; }>} options
+ * @returns {Promise<import("knex/types/tables").SubscriberRow[]>}
+ */
+// Not covered by tests; mostly side-effects. See test-coverage.md#mock-heavy
+/* c8 ignore start */
+async function getSubscribersWaitingForMonthlyEmail (options = {}) {
+  // I'm explicitly referencing the type here, so that these lines of code will
+  // show up as errors when we remove it from the flag list:
+  /** @type {import("./featureFlags.js").FeatureFlagName} */
+  const featureFlagName = "MonthlyActivityEmail";
+  const flag = await knex("feature_flags")
```

Comment:
Oh that's a good point, added that check in a util function in 1c425f71f.

The main reason I initially shied away from the util function, is because I had to know both `is_enabled` and the `allow_list`, and thus was already replicating feature flag-specific logic. I've now created a function that does a check for the `deleted_at`, but otherwise just returns the full row. We'll probably replace it by experiments later on anyway.